Elasticsearch tests requires jdk not jre

Elasticsearch jdk8 tests are failing as we are basing our test docker images from AdoptOpenJDK docker images that are jre not jdk builds.  Our images are using xxx but should maybe use for jdk11: https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/11/jdk/ubuntu/Dockerfile.openj9.nightly.full  (jdk11u-nightly tag)  
for jdk8: https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/8/jdk/ubuntu/Dockerfile.openj9.nightly.full  (jdk8u-nightly tag)

From [recent grinder output](https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1229/console):
23:32:57  Using docker image default Java
23:32:57  JAVA_BIN is: /opt/java/openjdk/bin
23:32:57  openjdk version "1.8.0_202"
23:32:57  OpenJDK Runtime Environment (build 1.8.0_202-b08)
23:32:57  Eclipse OpenJ9 VM (build openj9-0.12.1, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20190205_213 (JIT enabled, AOT enabled)
23:32:57  OpenJ9   - 90dd8cb40
23:32:57  OMR      - d2f4534b
23:32:57  JCL      - d002501a90 based on jdk8u202-b08)


23:37:15  FAILURE: Build failed with an exception.
23:37:15  
23:37:15  * What went wrong:
23:37:15  A problem occurred configuring project ':distribution'.
23:37:15  > jps executable not found; ensure that you're running Gradle with the JDK rather than the JRE
23:37:15  

Additionally, there was a workaround added at https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/thirdparty_containers/elasticsearch/dockerfile/elasticsearch-test.sh#L59-L63 for OpenJ9 prior to that implementation supporting jps tool.  That workaround may not actually be working, as seen by this output:
23:32:58  /elasticsearch-test.sh: line 61: /opt/java/openjdk/bin/jps: Permission denied
23:32:58  chmod: cannot access '/opt/java/openjdk/bin/jps': No such file or directory
but should no longer be needed as OpenJ9 now has the jps tool.


From [another Grinder output](https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1232/console), jdk11 Elasticsearch tests fail with:
02:28:39  FAILURE: Build failed with an exception.
02:28:39  
02:28:39  * What went wrong:
02:28:39  Could not determine java version from '11.0.2'.

Using nightly in my personal branch (also delete the workaround) got:

```
20:24:23  BUILD SUCCESSFUL in 11m 1s
20:24:23  473 actionable tasks: 473 executed
20:24:23  Elasticsearch Build - Completed
20:24:23  Running elasticsearch tests :
```
and 

```
20:34:16  JUnit4 test failed, ant output was:
20:34:16     [junit4] <JUnit4> says olá! Master seed: 4303BBF64C04856A
20:34:16     [junit4] JVM J0:     1.30 ..   261.08 =   259.77s
20:34:16     [junit4] JVM J1:     1.55 ..   264.43 =   262.89s
20:34:16     [junit4] JVM J2:     1.17 ..   261.52 =   260.34s
20:34:16     [junit4] Execution time total: 4 minutes 24 seconds
20:34:16     [junit4] Tests summary: 1017 suites (1 ignored), 6554 tests, 1 suite-level error, 80 ignored (80 assumptions)
20:34:16  
20:34:16  :core:test FAILED
20:34:16  
20:34:16  FAILURE: Build failed with an exception.
20:34:16  
20:34:16  * What went wrong:
20:34:16  Execution failed for task ':core:test'.
20:34:16  > There were test failures: 1017 suites (1 ignored), 6554 tests, 1 suite-level error, 80 ignored (80 assumptions) [seed: 4303BBF64C04856A]
20:34:16  
20:34:16  * Try:
20:34:16  Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
20:34:16  
20:34:16  * Get more help at https://help.gradle.org
```

https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1278/consoleFull

--- a/thirdparty_containers/elasticsearch/dockerfile/Dockerfile
+++ b/thirdparty_containers/elasticsearch/dockerfile/Dockerfile
@@ -24,9 +24,7 @@
  
 ARG SDK=openjdk8
 ARG IMAGE_NAME=adoptopenjdk/$SDK
-# Using nightly temporately util https://github.com/AdoptOpenJDK/openjdk-docker/issues/122 is fix
-#ARG IMAGE_VERSION=latest
-ARG IMAGE_VERSION=nightly
+ARG IMAGE_VERSION=latest
 FROM $IMAGE_NAME:$IMAGE_VERSION
 
 RUN groupadd -g 1000 elasticsearch && useradd elasticsearch -u 1000 -g 1000

